data_steward/analytics/cdr_ops/combined.py

```diff
@@ -285,6 +290,197 @@
 '''
 execute(client, query)
 
+# ## OMOP, Mapping, and Extension Tables Contain Same Record Count
```

Comment:
Perhaps add instructions for the check. Mainly mentioning that the results for both conditions are consolidated  in the output of the 'results' cell.

If it is possible at this time, add actions for the types of failure. Perhaps giving hints to the teams or process' that are responsible for creating these tables.

Reply:
Nice suggestions, I added some instructions. As for actions to take for different failure types, I'm not sure which processes/cleaning rules would cause the most issues.

@lrwb-aou Are there any specific cleaning rules are or other processes that would cause these failures to occur? If so, we can point notebook users in those directions.